<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="findMeButton">
          <property name="maximumSize">
           <size>
            <width>76</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Find me</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="recordButton">
          <property name="maximumSize">
           <size>
            <width>76</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Record</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="queryButton">
          <property name="maximumSize">
           <size>
            <width>76</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Query</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="settingsButton">
          <property name="maximumSize">
           <size>
            <width>76</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Settings</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exitButton">
          <property name="maximumSize">
           <size>
            <width>76</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWebEngineView" name="widget" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="positionLineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
